plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -1238,6 +1317,11 @@ public static void register() {
         // and the output max will contain the old min values.
         // Note that this is equivalent to constructing a cuboid with the input value and the original cuboids min value.
         // Not valid for multi-member CuboidTags.
+        // @example
+        // # Notes a new cuboid with the player's location as the new maximum location of the cuboid.
+        // # For example: if "my_cuboid" spans from 10,10,10 (max) to 5,5,5 (min) and the player had a location of 15,10,15,
+        // # then "my_new_cuboid" will have a minimum location of 15,10,15.
```

Comment:
Typo, min vs. max
Also for clarity's sake should probably rephrase this one and the above to more like `will span from 15,10,15 (max) to 5,5,5 (min)` (ie match the original phrasing in a way that emphasizes what changed and what hasn't)